.github/workflows/main.yml

```diff
@@ -1,12 +1,23 @@
+########################################################################################
+# The following secrets are required:
+#
+# 1. GH_ACCESS_TOKEN - A "fine-grained personal access token" generated through the
+#    Github UI. It seems like these tokens are scoped to a user, rather than an
+#    organisation.
+#
+#    The following minimum permissions are required:
+#      Read - access to metadata
+#      Read & write - access to actions and code
+# 2. GH_USER_NAME - The name (not username) associated with the Git user. e.g. John Smith
```

Comment:
I'm wondering if we should not use some dedicated "releasing" account for this. It feels a bit weird that the releases are done in the personal name.

Reply:
I think it's such a detail that is not worth changing. And the benefit is that it's going to point to the person owning the release (and likely the release token). We can move to a dedicated releasing account once there's need for it.